src/qpsolver/ratiotest.cpp

```diff
@@ -98,6 +98,7 @@ RatiotestResult ratiotest(Runtime& runtime, const Vector& p,
                           const Vector& rowmove, double alphastart) {
   switch (runtime.settings.ratiotest) {
     case RatiotestStrategy::Textbook:
+    default:  // to fix -Wreturn-type warning
```

Comment:
IIUC, `switch` statements don't look ahead. So, this case will be executed now for all subsequent cases as well. If it doesn't cause an error, it probably still means a slow down.